<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="log_request_num_lines" skipped="0" tests="1" time="32.42" timestamp="2022-07-22T07:24:34">
	<testcase classname="LogRequestVerifyNumberOfLines" name="log_request_num_lines">
		<failure message="BLOCKED: TypeError: 'NoneType' object is not subscriptable" type="BLOCKED"/>
		<system-out>07:24:02 INFO  ==============================================================
07:24:02 INFO  Id:    log_request_num_lines
07:24:02 INFO  Dir:   log_request_end_to_end/
07:24:02 INFO  Title: Validate end to end behaviour for the log request operation check for number of lines
07:24:02 INFO  ==============================================================
07:24:02 INFO  EnvironmentC8y Setup
07:24:02 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
07:24:02 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
07:24:02 INFO  Delaying execution by 7 seconds
07:24:23 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
07:24:24 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
07:24:24 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
07:24:25 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
07:24:25 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
07:24:26 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
07:24:27 INFO  Executed sudo&lt;log_file_daemon&gt;, exit status 0
07:24:27 WARN  TypeError occurred while running test - 'NoneType' object is not subscriptable
Traceback (most recent call last):
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py&quot;, line 1294, in __call__
    self.testObj.execute()
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/log_request_end_to_end/log_request_num_lines/run.py&quot;, line 46, in execute
    log_file_request_payload, self.project.deviceid
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/environments/cumulocity.py&quot;, line 101, in trigger_log_request
    &quot;deviceId&quot;: self.device_fragment[&quot;id&quot;],
TypeError: 'NoneType' object is not subscriptable
07:24:27 WARN  TypeError: 'NoneType' object is not subscriptable ... blocked 
07:24:27 INFO  
07:24:27 INFO  cleanup:
07:24:28 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
07:24:28 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
07:24:29 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
07:24:29 INFO  Executed sudo&lt;log_file_daemon&gt;, exit status 0
07:24:33 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
07:24:34 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
07:24:34 INFO  
07:24:34 INFO  Test duration: 32.42 secs
07:24:34 INFO  Test final outcome:  BLOCKED
07:24:34 INFO  Test outcome reason: TypeError: 'NoneType' object is not subscriptable
07:24:34 INFO  
</system-out>
	</testcase>
</testsuite>
